docs/getting-started/configure-app-with-plugins.md

```diff
@@ -34,23 +34,25 @@ CircleCI is a frontend UI plugin, it goes in `app` rather than `backend`.
 export { plugin as Circleci } from '@backstage/plugin-circleci';
 ```
 
-3. Register the plugin router:
+3. Register the plugin in the entity pages:
 
 ```jsx
 // packages/app/src/components/catalog/EntityPage.tsx
-
-import { Router as CircleCIRouter } from '@backstage/plugin-circleci';
+import {
+  EntityCircleCIContent,
+  isCircleCIAvailable,
+} from '@backstage/plugin-circleci';
 
 // Then somewhere inside <EntityPageLayout>
```

Comment:
Will be just `<EntityLayout>` now

Reply:
Clarified the example, because while you are spot on, in fact this particular example isn't even inside of an `<EntityLayout>`, so made it more specific to adding CI.